starter-code/src/viking.js

```diff
@@ -1,11 +1,76 @@
-// Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+
+  this.receiveDamage = function (damage){
+    this.health = this.health - damage;
+  };
+}
+
+Soldier.prototype.attack = function (){
+   return this.strength;
+  };
+
+
+  // Viking
+
+function Viking (name, health, strength) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+
```

Comment:
La gracia de la herencia es no tener que repetir las propiedades. Al hacer el Viking.prototype = Object.create(Soldier.prototype) lo que pasa es que vuestra función constructora Viking ya tiene por defecto las propiedades de Soldier. Solo teneis que añadir this.name = name.